mllib/src/main/scala/org/apache/spark/ml/feature/StringIndexer.scala

```diff
@@ -105,7 +125,11 @@ class StringIndexer @Since("1.4.0") (
 
 @Since("1.6.0")
 object StringIndexer extends DefaultParamsReadable[StringIndexer] {
-
+  private[feature] val SKIP_UNSEEN_LABEL: String = "skip"
+  private[feature] val ERROR_UNSEEN_LABEL: String = "error"
+  private[feature] val KEEP_UNSEEN_LABEL: String = "keep"
```

Comment:
It would make me even happier if these were public and could be used by the test code, but I think it's up to the commiters (jkbradley)